infra-files/init.d-unicorn_monitofen

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+
+### BEGIN INIT INFO
+# Provides:          unicorn
+# Required-Start:    $all
+# Required-Stop:     $all
+# Default-Start:     2 3 4 5
+# Default-Stop:      0 1 6
+# Short-Description: starts the unicorn app server
+# Description:       starts unicorn using start-stop-daemon
+### END INIT INFO
+
+set -e
+
+USAGE="Usage: $0 <start|stop|restart|upgrade|rotate|force-stop>"
+
+# app settings
+USER="pi"
+APP_NAME="monitofen"
+APP_ROOT="/home/$USER/$APP_NAME"
+ENV="production"
+
+# environment settings
+PATH="/home/pi/.rvm/gems/ruby-2.7.2/wrappers:$PATH"
+CMD="cd $APP_ROOT && bundle exec unicorn -c config/unicorn.rb -E $ENV -D"
+PID="$APP_ROOT/shared/pids/unicorn.pid"
+OLD_PID="$PID.oldbin"
+
+# make sure the app exists
+cd $APP_ROOT || exit 1
+
+sig () {
+  test -s "${PID}" && kill -$1 $(cat "${PID}")
```

Comment:
> __[ShellCheck]__ Double quote to prevent globbing and word splitting. ([view](https://sider.review/gh/repos/322395101/pulls/9#issue-5576932))
>
> <details>
>
> | Rule | Severity | Code |
> | ---- | -------- | ---- |
> | `SC2086` | `info` | 2086 |
>
> References:
> - <https://github.com/koalaman/shellcheck/wiki/SC2086>
>
> You can [close this issue](https://sider.review/gh/repos/322395101/pulls/9#issue-5576932) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

test commeny